MSAL/test/app/MSALStressTestHelper.m

```diff
@@ -44,14 +46,13 @@ @implementation MSALStressTestHelper
 
 + (void)expireAllTokensWithClientId:(NSString *)clientId
```

Comment:
Seems parameter clientId is not used and tokens for all clientIds will be expired ? :)

Reply:
Fixed